src/components/confirm/confirm.spec.js

```diff
@@ -111,17 +114,14 @@ describe("Confirm", () => {
             onCancel={onCancelFn}
           />
         );
-        domNode = wrapper.getDOMNode();
-        document.body.appendChild(domNode);
       });
 
       afterEach(() => {
-        document.body.removeChild(domNode);
         onCancelFn.mockReset();
       });
 
       it("should not close the modal if ESC key is pressed", () => {
```

Comment:
I think that we could change it to be what we are actually testing: `"onCancel" callback should not be called`